src/github.com/matrix-org/dendrite/mediaapi/routing/routing.go

```diff
@@ -48,9 +50,11 @@ func Setup(
 	}
 	authData := auth.Data{nil, deviceDB, nil}
 
+	// TODO: How to use tracing with these endpoints?
```

Comment:
I think this was essentially asking whether or not we need to use tracing with the DownloadAPI.

Reply:
Yeah, looks like it.